requirements.txt

```diff
@@ -58,6 +58,7 @@ geoip2==3.0.0
 gevent==1.4.0
 gipc==1.0.1
 greenlet==0.4.15
+grpcio==1.30.0
```

Comment:
Please add these two packages into alphabetical list in requirements-osbs.txt. Running `pipdeptree` indicates that their dependencies are already present (six and setuptools).

Reply:
done